app/models/room.rb

```diff
@@ -1,3 +1,4 @@
 class Room < ApplicationRecord
   belongs_to :hotel
-end
\ No newline at end of file
+  has_many :guests
```

Comment:
In order to add a `has_many :guests` association here on the `Room` model, that would mean that the `Guest` model would have a `belongs_to :room` association. But like we've learned, in order to have a `belongs_to :room` association, it means that there is a `room_id` foreign key on the guests table, which there is not. 

You could have a `has_many :guests` association here on the `Room` model, but it would need to go through the join table to get there.